lib/CMakeLists.txt

```diff
@@ -5,19 +5,19 @@
 
 
 file(GLOB LIB_SOURCES *.c)
-file(GLOB LIB_DI_SOURCES prot/di/*.c)
-file(GLOB LIB_TO1_SOURCES prot/to1/*.c)
-file(GLOB LIB_TO2_SOURCES prot/to2/*.c)
+#file(GLOB LIB_DI_SOURCES prot/di/*.c)
```

Comment:
why are these lines commented out? is this a WIP?

Reply:
Would be re-enabled in a subsequent PR.